pallets/liquidity-pools/src/message.rs

```diff
@@ -226,7 +226,7 @@ pub enum Message<BatchContent = BatchMessages> {
 		/// The hash of the message which shall be recovered
```

Comment:
As of now, these are only used for outbound - Centrifuge -> Evm. Do we also expect these for inbound - Evm -> Centrifuge? @hieronx

Reply:
That is right, `Initiate/DisputeMessageRecovery` are only Centrigue -> EVM